_authors/omair-javaid.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Omair Javaid
+first_name: Omair
+last_name: Javaid
+username: Omair.Javaid@linaro.org
+image: /assets/images/content/omair-javaid.webp
```

Comment:
Could you provide a jpg/png for this image? We are generating webp versions of images when the site is built.

BR,

Kyle

Reply:
Have requested this image file.